spec/unit/actions/app_delete_spec.rb

```diff
@@ -276,19 +276,15 @@ module VCAP::CloudController
               let!(:binding1) { ServiceBinding.make(app: app, service_instance: ManagedServiceInstance.make(space: app.space)) }
 
               it 'should always call the broker with accepts_incomplete true' do
-                expect(client).to receive(:unbind).with(binding1, user_audit_info.user_guid, true)
+                expect(client).to receive(:unbind).with(binding1, nil, true)
```

Comment:
Hi @Gerg, @blgm and I also noticed this while working on this PR. We hadn't realise v2 was taking this value mostly out of the `SecurityContext` and we needed to pass as a parameter for `jobs` as there is no `SecurityContext` from within them. So we are currently fixing that for all services `jobs`. It will be fixed when rebasing on main.

Reply:
After rebasing, https://github.com/cloudfoundry/cloud_controller_ng/blob/beaf77b273716ebbfb9b1996f20924486b115bfd/spec/unit/actions/app_delete_spec.rb#L279